src/metrics/function_error.py

```diff
@@ -37,6 +37,28 @@ def evaluate(self, models: Iterable[ContextModel], num_batches: int = 1) -> Iter
 
         return errs
 
+class RegressionScore(Benchmark):
+    # zero_err is of shape (1)
+    # optimal_err is of shape (seq_length,)
+    def __init__(self, metric: Metric, function_class: FunctionClass, zero_err, optimal_err):
+        self.function_class = function_class
+        self.metric = metric
+        self.zero_err = zero_err
+        self.optimal_err = optimal_err.mean(dim = 1)
+
+    def evaluate(self, models: Iterable[ContextModel]) -> Iterable[Tensor]:
```

Comment:
This should return an iterable of `float`s. A 1-D Tensor (as you already have) satisfies this type constraint

Reply:
Wouldn't changing this to Iterable(float) cause an error due to overwriting the super class's (Benchmark) evaluate incorrectly?

Reply:
it would, my bad